vue-vuetify/src/complex/OneOfRenderer.vue

```diff
@@ -123,14 +123,15 @@ const controlRenderer = defineComponent({
     const newSelectedIndex = ref(0);
     const dialog = ref(false);
     const t = useTranslator();
-
+    const { handleBlur } = useBlurHandler(control);
```

Comment:
The `useBlurHandler` could also be made part of `useVuetifyControl`, then we wouldn't need to call it in every `setup` method.

Reply:
Good idea! Moved `handleBlur` into `useVuetifyControl`.